Update devilbox.en.md

Ernestopheles opened the pull request:
I edited the markup file in Vscode and used a markup preview extension

Ernestopheles committed:
Update devilbox.en.md

fritzmg reviewed (changes requested):
Please revert any unrelated changes.

Ernestopheles committed:
1 correction and some reverts also

Thanks, @fritz for review

Ernestopheles committed:
for test

for test hugo

Ernestopheles committed:
Correction version infos

exchanged use of {{ version }} with plain text, as  Devilbox version is not ment there.

Ernestopheles commented:
closed as of #1137
